scripts/check-ai-catalog-inclusion.sh

```diff
@@ -0,0 +1,214 @@
+#!/bin/bash
+
+# Script to check that all Docker Hub AI namespace repositories are included in the Gen AI catalog
+# Usage: ./check-ai-catalog-inclusion.sh --namespace <namespace>
+
+set -euo pipefail
+
+# Default values
+NAMESPACE=""
+
+# Function to show usage
+show_usage() {
+    cat << EOF
+Usage: $0 --namespace <namespace>
+
+Check that all Docker Hub AI namespace repositories are included in the Gen AI catalog.
+
+Required Arguments:
+  --namespace <name>    Docker Hub namespace to check (must be 'ai')
```

Comment:
Why the required flag with only one argument allowed?

Reply:
Added [here](https://github.com/docker/model-distribution/pull/105/commits/e8ac139b695f73fdaf60ff2cf00b51d05ca85563) and also updated the logs to use `$NAMESPACE` instead of hardcoding `AI`